app/javascript/pages/admin/Organizations/index.js

```diff
@@ -27,14 +27,37 @@ const actionLinks = (organization, togglePopover) => (
   </div>
 )
 
+const websiteLink = ({ original: organization }) => {
+  let websiteUrl = organization.website
+  if (!/^https?:\/\//i.test(websiteUrl)) {
+    websiteUrl = 'http://' + websiteUrl
```

Comment:
I would rather have the server ensure a valid URL gets served. Maybe we put a validator on the organization model and a handler to return a nice and correctly formatted URL.